### PR TITLE
added PostgresSQL settings for max_idle and max_open connections

### PR DESCRIPTION
**Pull request by kimmokh**
This PR would solve issue #356 - _Add settings for limiting PostgreSQL MaxOpenConns and MaxIdleConns_

### CODE REVIEW

**Comment by brocaar** [comment]
Awesome,. thanks! :+1: Would you mind making the same change for the loraserver repository?

File: cmd/lora-app-server/cmd/configfile.go [hunk 1]
```diff
@@ -62,6 +62,13 @@ dsn="{{ .PostgreSQL.DSN }}"
 # App Server and / or applying migrations.
 automigrate={{ .PostgreSQL.Automigrate }}
 
+# Max open connections in the pool.
+# <= 0 means unlimited
+max_open={{ .PostgreSQL.MaxOpen }}
```
Comment (brocaar) [inline cmd/lora-app-server/cmd/configfile.go]: One comment, would you mind renaming this to `max_open_connections` (and `MaxOpenConnections`) and `max_idle_connections` (and `MaxIdleConnections`)? That way the configuration variables are a bit more descriptive. After that I'll merge your changes.

Comment (kimmokh) [reply cmd/lora-app-server/cmd/configfile.go]: Fields renamed.

Comment (kimmokh) [reply cmd/lora-app-server/cmd/configfile.go]: Pull request for LoRaServer here: [loraserver pr 437](https://github.com/brocaar/loraserver/pull/437)

**Pull request merged**